akirachix/Magezi-Informational
# Landing page

# Engineers Checklist

Please include a summary of the change and which issue is fixed. 
This pull request (PR) refines the footer design to align with the layout of the landing page, ensuring a consistent user experience across the application. 


## What is this?
* This PR modifies the footer component to implement a flexbox layout similar to the landing page, enhancing visual consistency.

##Why this?
* The change addresses user feedback regarding layout inconsistencies and aims to improve the overall aesthetic and usability of the application.

## How are you doing this? 
* By adjusting the layout to use flexbox, modifying spacing, and aligning text, this change ensures the footer matches the design  in the landing page.

##Dependencies
No new dependencies are required for this change. It utilizes existing styles and components within the application.


## Type of change


## How can it be tested?
-Visual Inspection
Responsive Design-Test the footer on different screen sizes (mobile, tablet, desktop) to ensure proper alignment and spacing.
Functionality Check-Click on links in the footer to confirm they are operational and lead to the expected pages.
